### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by matias200308** [0x1317bf7/PodClassic#4]
Allow selecting device folders, to avoid mixing music with WhatsApp audios among other files on the device

**Comment by K4TieK** [comment]
I agree, a more simplified folder selection we can pick would be much more enjoyable!!
